docs/02.API-REFERENCE.md

```diff
@@ -11924,3 +11924,76 @@ main (void)
   return 0;
 }
 ```
+
+# Container Functions
+
+## jerry_get_buffer_from_container
+
+**Summary**
+
+Return a new array containing elements from a Container, or a Container Iterator. Sets the boolean input value to `true`
+if the container object contains key-value structure and false if not.
+
+*Notes*
+- This API function depends on a build option (`JERRY_BUILTIN_CONTAINER`) and can be checked
+  runtime with the `JERRY_FEATURE_MAP, JERRY_FEATURE_SET, JERRY_FEATURE_WEAKMAP, JERRY_FEATURE_WEAKSET`
+  feature enum values.
+  see: [jerry_is_feature_enabled](#jerry_is_feature_enabled).
+- The es.next profile enables this by default.
+
+*New in version [[NEXT_RELEASE]]*.
+
+**Prototype**
+
+```c
+jerry_value_t
+jerry_get_buffer_from_container(jerry_value_t value,
+                                bool* is_key_value_p)
+```
+
+- `value` - Container object
+- `is_key_value` - Will be set to `true` if the given container has key-value pairs, `false` otherwise.
+- return value
+  - jerry_value_t containing an array of values of the underlying buffer, if the orignal is not empty
```

Comment:
What is the original? I think the description is too long, this could go to the notes, and `- jerry_value_t containing an array of values` could be enough here